services/web/package.json

```
@@ -9,10 +9,12 @@
     "build": "npm run build:remix && npm run build:ts",
     "dev": "remix dev -c \"nodemon\"",
     "lint": "eslint --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint .",
-    "start": "node ./dist/server.js"
+    "start": "node ./dist/server.js",
+    "million": "npm run million"
```

```suggestion
    "start": "node ./dist/server.js"
```

no need for this anymore